packages/neuron-wallet/src/services/syncBlocks.ts

```diff
@@ -31,18 +40,33 @@ export default class SyncBlocksService {
 
   private fetchSize = 128
 
-  private minFetchSize = 2
-
-  private sizeForCheck = 5
+  private minFetchSize = 16
```

Comment:
Now that we have `lines-between-class-members` eslint rule configured, I guess we can remove some blank lines between related members(properties).

Reply:
updated